### ISSUE TITLE
authentication required (fingerprint and/or pin) is repeated endless

### ISSUE THREAD

**Issue by micressor** [digitalbitbox/bitbox-wallet-app#2449]
I have just activated remember wallet function.

It is currently difficult to open the app at all. It takes several attempts
* clear cache
* Close/Kill app

until I can open it. Is this a known problem?

* Version: [v4.40.0](https://github.com/digitalbitbox/bitbox-wallet-app/releases/tag/v4.40.0)
* LineageOS without google play services

**Comment by Beerosagos** [comment]
You enabled the screen lock, right? If you disable it does everything work as expected?

**Comment by micressor** [comment]
@Beerosagos Exactly, when I disable screen lock it works.

**Comment by micressor** [comment]
@Beerosagos If needed, I can take my time and provide logs.

**Comment by Beerosagos** [comment]
> @Beerosagos If needed, I can take my time and provide logs.

Yeah, if you can that could be helpful. We don't explicitly support android-based platforms different from Android itself, as it would be hard to test and reproduce issues (just to manage expectations). But if you can provide a log I can give it a shot.
